testing/php-couchbase/APKBUILD

```diff
@@ -1,62 +0,0 @@
-# Contributor: Nathan Johnson <nathan@nathanjohnson.info>
-# Maintainer: Nathan Johnson <nathan@nathanjohnson.info>
-pkgname=php-couchbase
-_pkgname=couchbase
-pkgver=2.2.3
-pkgrel=1
-pkgdesc="PHP extension couchbase"
-url="http://pecl.php.net/package/$_pkgname"
-arch="all"
-license="ASL-2.0"
-depends=""
-makedepends="autoconf libcouchbase-dev zlib-dev"
-subpackages=""
-source="http://pecl.php.net/get/$_pkgname-$pkgver.tgz"
-builddir="$srcdir/$_pkgname-$pkgver"
-
-_phpversions="5 7"
-for _v in $_phpversions; do
-	makedepends="$makedepends php$_v-dev"
-	subpackages="$subpackages php$_v-${pkgname#php-}:_subpackage"
-done
-
-prepare() {
-	default_prepare || return 1
-
-	local ver; for ver in $_phpversions; do
-		cp -ar "$builddir" "$builddir-$ver"
-	done
-}
-
-build() {
-	local ver; for ver in $_phpversions; do
-		cd "$builddir-$ver"
-
-		phpize$ver || return 1
-		./configure --prefix=/usr \
-			--with-php-config=/usr/bin/php-config$ver || return 1
-		make || return 1
-		make test || return 1
-	done
-}
-
-package() {
-	mkdir -p "$pkgdir"
-}
-
-_subpackage() {
-	local ver="${subpkgname:3:1}"
-	pkgdesc="$pkgdesc (for PHP $ver)"
-	depends="$depends php$ver"
-	install_if="$pkgname=$pkgver-r$pkgrel php$ver"
-	[ "$ver" = 5 ] && replaces="$pkgname"
-
-	make -C "$builddir-$ver" INSTALL_ROOT="$subpkgdir" install || return 1
-
-	install -d "$subpkgdir"/etc/php$ver/conf.d || return 1
-	echo "extension=$_pkgname.so" > "$subpkgdir"/etc/php$ver/conf.d/$_pkgname.ini
-}
-
-md5sums="f58ee1596ade978832f033f2e1662579  couchbase-2.2.3.tgz"
-sha256sums="5c998e9c98e51a783a5d74ec36aa6ac740a4a958dd8c206ed9e9d1580234907f  couchbase-2.2.3.tgz"
-sha512sums="d1feeb33a8ce450919731d17f84b170eaa68da104cedfe8ffa56f4e2e6062e250077d00babaca4872cc5bf271509daee2ca83e35cfb4b8ceef4b402cc20c1bf3  couchbase-2.2.3.tgz"
```

Comment:
because it fails to build in travis-ci
apk add php5-dev php7-dev raise an error about conflict in /usr/bin

`$ apk add php5-dev php7-dev ; echo $?

(1/7) Installing php5-config (5.6.30-r1)
(2/7) Installing libxml2 (2.9.4-r1)
(3/7) Installing php5 (5.6.30-r1)
(4/7) Installing php5-dev (5.6.30-r1)
(5/7) Installing php7-config (7.1.3-r0)
(6/7) Installing php7 (7.1.3-r0)
ERROR: php7-7.1.3-r0: trying to overwrite usr/bin/php owned by php5-5.6.30-r1.
(7/7) Installing php7-dev (7.1.3-r0)
ERROR: php7-dev-7.1.3-r0: trying to overwrite usr/bin/phpize owned by php5-dev-5.6.30-r1.
ERROR: php7-dev-7.1.3-r0: trying to overwrite usr/bin/php-config owned by php5-dev-5.6.30-r1.
Executing busybox-1.26.2-r0.trigger
2 errors; 414 MiB in 164 packages
2
$`